src/reactors/RelayOrderReactor.sol

```diff
@@ -34,13 +32,12 @@ contract RelayOrderReactor is ReactorEvents, ReactorErrors, ReentrancyGuard, IRe
         permit2 = _permit2;
     }
 
+    // write execute such that we prepare,execute, and fill per order
     function execute(SignedOrder calldata order) external payable nonReentrant {
```

Comment:
We should also be able to remove payable from these functions right? Since there are no output tokens, fillers never need to send ETH to this contract